### PR TITLE
CCP-138 Bean included for concept add screen

### PR DESCRIPTION
**Pull request by mkarthik90**
Added bean for concept add screen

### CODE REVIEW

**Comment by mkarthik90** [comment]
I was not able to reopen the closed PR. Because I squashed the commits and did a force push.

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by mkarthik90** [comment]
@jdamerow Rebased

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import java.util.ArrayList;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java]: indentation seems to be off in this file

File: Conceptpower+Spring/src/main/resources/validatormessages_en_US.properties [hunk 2]
```diff
@@ -14,27 +14,28 @@ match.passwords = Passwords do not match.
 concept_name.required = Concept name is mandatory.
 concept_description.required = Concept Description is mandatory.
 concept_unique.required = Concept name should be unique.
-concept_name.wordnet = Concept name should not be WordNet.
+concept_list_name.wordnet = Concept list name can not be WordNet.
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/resources/validatormessages_en_US.properties]: "can not" should be one word

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptAddView.jsp [hunk 3]
```diff
@@ -1,566 +1,611 @@
...
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptAddView.jsp]: let's loose a bunch of those empty lines in this file. Only put them where they make sense, and then maybe one.

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java [hunk 4]
```diff
@@ -50,111 +55,134 @@
...
+     * @return returns string which redirects to concept creation page
+     */
+    @RequestMapping(value = "auth/conceptlist/addconcept")
+    public String prepareConceptAdd(ModelMap model, @ModelAttribute("conceptAddBean") ConceptAddBean conceptAddBean) {
+
+        // model.addAttribute("ServiceBackBean", new ServiceBackBean());
+        Map<String, String> serviceNameIdMap = serviceRegistry.getServiceNameIdMap();
+        conceptAddBean.setServiceNameIdMap(serviceNameIdMap);
+        // model.addAttribute("SearchResultBackBeanForm", new
+        // SearchResultBackBeanForm());
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java]: remove those commented out lines

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java (hunk 1 above)
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java]: Do I need to reformat the file? I am not sure what to change

Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/wrapper/impl/ConceptEntryWrapperCreator.java]: yes. if you look through the code you will see the indentation is off.

**Comment by mkarthik90** [comment]
@jdamerow  Fixed review comments

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java [hunk 5]
```diff
@@ -206,7 +230,7 @@ public String addConcept(HttpServletRequest req, Principal principal, ModelMap m
         try {
             entries = conceptManager.getConceptListEntriesForWord(conceptname.trim());
         } catch (IndexerRunningException e1) {
-            return new ResponseEntity<String>(e1.getMessage(),HttpStatus.CONFLICT);
+            return new ResponseEntity<String>(e1.getMessage(), HttpStatus.CONFLICT);
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java]: see all my other comments. I think conflict is not the right error code here.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
please fix tests

**Pull request merged**